### PR TITLE
Feature/create climber subsystem

### LINKED ISSUE #19: Create Climber subsystem
Create the climber subsystem.

Should be driven by 1 motor. Method needed for setting speed.

When initialized (and always) motor should be in Brake mode.

### PR DESCRIPTION
**Pull request by citrus-melon**
Closes #19 

This code creates the climber subsystem and creates a method to set the speed of the climber.

### CODE REVIEW

**Review by chit-sudeep — DISMISSED** [review]
Looked over all of the climber subsubsytem code and its javadocs. 
Looks good!

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Climber extends SubsystemBase {
+  private final CANSparkMax climberMotor;
+
+  /**
+   * Creates a Climber with its {@link CanSparkMax} at the given CAN id.
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/subsystems/Climber.java]:
```suggestion
   * Creates a Climber with its {@link CANSparkMax} at the given CAN id.
```

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   * 
+   * @param motorId The CAN id of the motor
+   */
+  public Climber(int motorId) {
+    this(motorId, MotorType.kBrushless);
+  }
+
+  /**
+   * Creates a Climber with its {@link CanSparkMax} at the given CAN id.
```
Comment (aidan-mundy) [inline src/main/java/frc/robot/subsystems/Climber.java]:
```suggestion
   * Creates a Climber with its {@link CANSparkMax} at the given CAN id.
```

**Review by aidan-mundy — CHANGES_REQUESTED** [review]
Mostly good, just needs a couple of capitalization changes (as mentioned in my suggestions). Let me know when these have been made and I'll approve!

**Review by chit-sudeep — APPROVED** [review]
Looked over all of the climber subsubsytem code, its javadocs, and checked if the capitalization mistakes were fixed.
Looks good!

**Review by aidan-mundy — APPROVED** [review]

**Pull request merged**